config/initializers/huginn_agent.rb

```diff
@@ -0,0 +1,5 @@
+require 'huginn_agent'
+
+HuginnAgent.hack_huginn_to_accept_me
```

Comment:
@cantino Yeah, that would be fine as well, and would remove the need for the monkey-patch.